src/components/expansionPanel.js

```diff
@@ -82,6 +84,8 @@
       expandIcon: <Icon name="ExpandMore" />,
     };
 
+    const ShowChilderen = <>{dataShown || isDev ? children : ''}</>;
```

Comment:
Are the fragments necessary here?

Reply:
I think so, otherwise I had a validation error